src/OrbitGl/App.h

```diff
@@ -114,14 +116,21 @@ class OrbitApp final : public DataViewFactory,
                                                            const std::filesystem::path& dest);
   void OnLoadCaptureCancelRequested();
 
-  [[nodiscard]] orbit_capture_client::CaptureClient::State GetCaptureState() const;
+  // vvv AppCaptureClientInterface
```

Comment:
I guess this will be removed later?

Reply:
I think this is supposed to be a section comment. (See the other comment below `// ^^^`)

Since this was confusing to at least one reviewer and since we don't use this style anywhere else in the code base, maybe we should either remove it or replace it by dashes `// ----------`? WDYT?

I'm undecided. Of course we shouldn't need sections because our classes are small and have only one purpose. :smile: 
But it's also clear we won't fix that in this PR. :innocent:

Reply:
😅 I agree, I'm not a fan of the sections either, but it's also confusing to have this large class implementing multiple interfaces without any grouping... I'll replace it with dashes for the time being.